uiowa/uiowa
# Untangling Events and Admissions config

This is hopefully the first step in undoing some of the fragmentation that had occurred between the Admissions site split and the event feature.

1. Removes the full split of `views.view.events` from admissions.
2. Removes layout split from admissions.
3. Auto-submit is turned on for the default /events view.
4. The subevents view on the full node page has the "No results" removed so that nothing will appear if there are no subevents. 
5. The /events/calendar view is disabled for the feature split but turned on for admissions only. 
6. The local tasks css was moved to a separate library and is loaded separate for the local tasks block so that it can be used for the "Tabs" that are unstyled at https://admissions.uiowa.edu/events. 


# How to test

`ddev blt frontend`

```
ddev blt ds && ddev drush @default.local config-split:activate event -y && ddev drush @default.local cim -y && ddev drush @default.local uli events
```
- Check default + event feature
```
ddev blt ds --site admissions.uiowa.edu && ddev drush @admissions.local uli events
```
- Check Admissions and compare local /events page with prod /events page. 
```
ddev blt ds --site iowasummerwritingfestival.uiowa.edu && ddev drush @iowasummerwritingfestival.local uli events
```
- Check ISWF and compare local /events page with prod /events page. 
```
ddev blt ds --site oniowa.uiowa.edu && ddev drush @oniowa.local uli events
```
- Check OnIowa! and compare local /events page with prod /events page. 

## How to test views exposed form design updates
### Check Admissions
```
ddev blt ds --site admissions.uiowa.edu && ddev drush @admissions.local uli areas-of-study
```

- Test desktop/mobile breakpoints
- Test search and reset buttons
- Compare https://admissions.uiowa.ddev.site/areas-of-study against https://admissions.uiowa.edu/areas-of-study
- Compare https://admissions.uiowa.ddev.site/events against https://admissions.uiowa.edu/events (Upcoming and Calendar tabs are now styled for anonymous users)

### Check Housing
```
ddev blt ds --site housing.uiowa.edu && ddev drush @housing.local uli residence-halls
```
- Test desktop/mobile breakpoints
- Test search and reset buttons
- Compare https://housing.uiowa.ddev.site/residence-halls against https://housing.uiowa.edu/residence-halls
